frontend/app/src/components/common/icon_button.tsx

```diff
@@ -132,9 +133,9 @@ export default function IconButton(props: any) {
         paddingRight: props.leadingIcon && 10,
         ...props.style
       }}
-      hovercolor={props.hoverColor ?? hexToRgba(colors[color].background, 0.8)}
-      activecolor={props.activeColor}
-      shadowcolor={props.shadowColor}
+      hovercolor={props.hovercolor ?? hexToRgba(colors[color].background, 0.8)}
+      activecolor={props.activecolor}
+      shadowcolor={props.shadowcolor}
```

Comment:
why did the names change here? are they used like this everywhere IconButton is used?

Reply:
closing cause looks like hoverColor is never used